pip-req.txt

```diff
@@ -11,4 +11,5 @@ gensim>=0.11.1
 pyparsing>=2.0.3
 twython>=3.2.0
 regex>=2019.08.19
-
+pytest>=6.0.1
```

Comment:
Why is pytest in both these requirements and in `requirements-test.txt`?

Reply:
The way this file has been written, it seems to define the versions of _every_ pip package (nose and coverage are here too). I think your point is good, and I moved the pytest packages (and their versions) to `requirements-test.txt`